# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: target/mnemosyne/overlay.d/sysfiles-embedded/etc_passwd.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./etc/passwd.orig	2008-09-22 15:06:42.000000000 +0000
+++ ./etc/passwd	2008-09-22 15:07:56.000000000 +0000
@@ -1,5 +1,4 @@
-root:x:0:0:root:/root:/bin/bash
-toor:x:0:0:emergency root:/root:/bin/kiss
+root:x:0:0:root:/root:/bin/sh
 bin:x:1:1:bin:/bin:/bin/false
 daemon:x:2:2:daemon:/sbin:/bin/false
 nobody:x:65534:65534:nobody:/var/empty:/bin/false
